packages/client/src/Client.ts

```diff
@@ -51,11 +51,15 @@ export interface Client<O extends OntologyDefinition<any>>
 // Once we migrate everyone off of using the deprecated parts of `Client` we can rename this to `Client`.
 export interface FutureClient {
   <
-    Q extends ObjectOrInterfaceDefinition | ActionDefinition<any, any, any>,
+    Q extends
```

Comment:
nit: Seems to be a lot of places where the type generic signatures got reformatted, it's fine but just flagging if this was unintentional